### PR TITLE
[CAGX] Driver update fix

### PR DESCRIPTION
**Pull request by agirault**
Preview: https://github.com/nvidia-holoscan/holoscan-docs/blob/ag/fix-driver-update-cagx/devkits/clara-agx/clara_agx_user_guide.md#update-nvidia-drivers

____

### 1. Instructions mixed install from local repo + remote repo

stick to remote repo only

____

### 2. Current instructions do not work:

```sh
$ sudo apt install cuda-drivers-550
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 cuda-drivers-550 : Depends: nvidia-driver-550 (>= 550.90.12) but it is not going to be installed or
                             nvidia-driver-550-open (>= 550.90.12) but it is not going to be installed or
                             nvidia-driver-550-server (>= 550.90.12) but it is not going to be installed or
                             nvidia-driver-550-server-open (>= 550.90.12) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Probably worked because it was the latest version at the time it was written

Fix by updating instructions to install `cuda-drivers` instead (which can be versioned)

____

### 3. Add instructions for download open kernel flavors

Was tricky due to removal of nvidia-l4t BSP packages with [regular instructions](https://docs.nvidia.com/datacenter/tesla/driver-installation-guide/#switching-between-driver-module-flavors), as well as version mismatch for 535 due to open package hosted on canonical only.

### CODE REVIEW

File: devkits/clara-agx/clara_agx_user_guide.md [hunk 1]
```diff
@@ -606,22 +606,38 @@ For feedback, discussion, and questions, please post to the Clara Holoscan SDK [
...
+>
+> ```sh
+> # Example for 560 drivers
+> sudo apt install cuda-drivers="560.*" nvidia-driver-560-open
+> sudo reboot
+> ```
+> 
+> For `535` that is even more tricky, as the CUDA APT repository does not include the `nvidia-driver-535-open` package. It is available from Canonical/Ubuntu servers instead, but with strict dependencies versions that precede what the NVIDIA repository provides (`535.183.01-0ubuntu0.20.04.1` instead of `535.183.06-0ubuntu1`) and APT cannot resolve it automatically and will error out with `Unmet dependencies` when running the command above. If you must use `535-open`, explicitly add all the dependencies listed by `Unmet dependencies / Depends:` suffixed with `=="535.183.01-0ubuntu0.20.04.1"` to your `apt install` command.
+
+### Validate Version
```
Comment (tbirdso) [inline devkits/clara-agx/clara_agx_user_guide.md:655]: Could you provide sample output? Probably straightforward 535 driver output, but good to include.

**Review by tbirdso — APPROVED** [review]
Thanks @agirault , clear and to the point. Would be good to have someone else with CAGX access test the upgrade as well.

File: devkits/clara-agx/clara_agx_user_guide.md (hunk 1 above)
Comment (agirault) [reply devkits/clara-agx/clara_agx_user_guide.md:655]: of `nvidia-smi` or `nvidia-smi -i 0 --query-gpu=driver_version --format=csv,noheader`?

**Comment by agirault** [comment]
@okutter @brudfors I might want for one of you to review/test before I squash and merge this. ty!

**Comment by brudfors** [comment]
@agirault I have tried the instructions to upgrade to 560 driver on my CAGX (currently running 550.90.07). 

Installing the CUDA APT keyring works but:
```sh
sudo apt install cuda-drivers="560.*" nvidia-driver-560-open
```
gives me:
```sh
sudo apt install cuda-drivers="560.*" nvidia-driver-560-open
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Version '560.*' for 'cuda-drivers' was not found
E: Unable to locate package nvidia-driver-560-open
```

**Comment by agirault** [comment]
@brudfors had an issue where he configured the iGPU keyring instead of the dGPU keyring. iGPU keyring had to be uninstalled to install the dGPU one.

Ran into another issue afterwards due to his old install of 550. Added a commit to document the known limitation and workaround.

**Pull request merged**